src/main/java/mate/academy/spring/mapper/impl/response/UserResponseMapper.java

```diff
@@ -0,0 +1,17 @@
+package mate.academy.spring.mapper.impl.response;
+
+import mate.academy.spring.mapper.DtoResponseMapper;
+import mate.academy.spring.model.User;
+import mate.academy.spring.model.dto.response.UserResponseDto;
+import org.springframework.stereotype.Component;
+
+@Component
+public class UserResponseMapper implements DtoResponseMapper<UserResponseDto, User> {
```

Comment:
Consider renaming the class to `UserToUserResponseDtoMapper` to make its purpose more clear.

Reply:
Whether it is necessary to apply this approach in naming for other mappers?